src/api/config/initializers/influxdb_dj_subscriber.rb

```diff
@@ -0,0 +1,40 @@
+if CONFIG['influxdb_hosts']
+  ActiveSupport::Notifications.subscribe(/active_job/) do |*args|
+    event = ActiveSupport::Notifications::Event.new(*args)
+    job = event.payload[:job]
+    exception = event.payload[:exception_object]
+
+    def write_queue_stats(state, job)
+      puts "write_queue_stats #{state}"
+      InfluxDB::Rails.client.write_point('active_job.queue',
+                                         tags: { state: state, job: job.class.name, queue: job.queue_name },
+                                         values: { value: 1 })
+    end
+
+    def write_performance_stats(job, event)
+      puts "write_performance_stats #{event.duration}"
+      InfluxDB::Rails.client.write_point('active_job.performance',
+                                         tags: { job: job.class.name, queue: job.queue_name },
+                                         values: { value: event.duration })
+    end
+
+    if event.name == 'perform.active_job' && exception
+      InfluxDB::Rails.client.write_point('active_job.queue',
+                                         tags: { state: :failed, job: job.class.name, queue: job.queue_name, id: job.job_id },
+                                         values: { value: 1 })
+      return true
+    end
+
+    case event.name
+    when 'enqueue.active_job'
+      write_queue_stats(:queued, job)
+    # we do not use this currently, we use clockwork instead...
```

Comment:
I don't think this comment is still true

Reply:
Why?

Reply:
ok, let me rephrase: the comment is very confusing. I bet there are a lot of other things not used you can put here as comments - just to confuse reviewers :)

Reply:
better?